### PR TITLE
163 rename all charegeing station references to charege point

### PR DESCRIPTION
**Pull request by WinterBoii**
## Description
Changed station to chargerpoint

## Screenshots
Add if necessary.
Copy paste should work.

## Type of PR
- [ ] Feature
- [ ] Bugfix
- [ ] Documentation
- [ ] Test
- [ ] Code refactor

## Did you write tests?
- [ ] Yes
- [ ] No
- [ ] No, I need help

## If you wrote tests, did they pass?
- [ ] Yes
- [ ] No
- [ ] No, I need help

## In which environment did you test?
- [ ] Development
- [ ] Production (backend API)
- [ ] Didn't test

### example steps (change for your PR):
- [ ] Click on the Active Charger Points button
- [ ] Make sure that its Charger Points

### CODE REVIEW

**Comment by eejebring** [comment]
I seem to experience some type errors like ```stations.location is undefined```
And also the variable should be named chargePoint... if you have time.

**Comment by vikjoh1** [comment]
@eejebring I believe these errors you get has to do with that backend changed the name of ```location``` to ```coordinates```. 
I got the same errors in any branch yesterday. However, it has been changed back to ```location``` again, so try to remove the docker container and then build it again.

**Review by vikjoh1 — CHANGES_REQUESTED** [review]
Looks good! However, you have missed to rename it in some places. if you ```ctrl+shift+f``` in VS code and search "charger station" you will see where it is ;)

**Review by Ville-Wilson — APPROVED** [review]
LGTM! 👍

**Comment by eejebring** [comment]
image
only the edit modal left from what I can see

**Review by vikjoh1 — CHANGES_REQUESTED** [review]
Is it ```charge-point``` or ```chargePoint```? Because it's named different in some components. Also, when adding a charge point, ```station price``` should be ```charge-point price``` and ```station name``` should be ```charge-point name```?

image

And personally, I think ```add point``` should be changed to ```add charge-point```

**Comment by eejebring** [comment]
...Cannot find any UI place where I can see the Charge-points named ChargePoint

File: react-app/src/components/pages/chargers/page-components/AddChargerDialog.tsx [hunk 1]
```diff
@@ -99,29 +99,29 @@ const AddChargerDialog: FC<AddChargerDialogProps> = ({ open, handleClose, statio
           Chager: {state.successfulAddedSerialNumber} Added
         </Alert>
       </Collapse>
-      <DialogTitle>Add Chargers to a station</DialogTitle>
+      <DialogTitle>Add Chargers to a chargePoint</DialogTitle>
```
Comment (vikjoh1) [inline react-app/src/components/pages/chargers/page-components/AddChargerDialog.tsx:102]: here :)

image

**Review by vikjoh1 — CHANGES_REQUESTED** [review]

**Pull request merged**